daemon/algod/server.go

```diff
@@ -185,7 +186,12 @@ func (s *Server) Start() {
 	}
 
 	addr = listener.Addr().String()
-	server = http.Server{Addr: addr, Handler: handler}
+	server = http.Server{
+		Addr:         addr,
+		Handler:      apiHandler,
+		ReadTimeout:  15 * time.Second,
```

Comment:
Will this break the wait-for-block-after call that waits up to 1 minute?

Reply:
@Karmastic - Good find, yes, this timeout would trigger before the wait-for-block timeout. I'll raise the default timeout.